src/middlewared/middlewared/plugins/etc.py

```diff
@@ -297,6 +297,7 @@ class EtcService(Service):
                 {'type': 'mako', 'path': 'pam.d/sshd', 'local_path': 'pam.d/sshd_linux'},
                 {'type': 'mako', 'path': 'local/users.oath', 'mode': 0o0600, 'checkpoint': 'pool_import'},
                 {'type': 'py', 'path': 'local/ssh/config'},
+                {'type': 'mako', 'path': 'login_banner', 'mode': 0o600},
```

Comment:
Also need to add `system.advanced.login_banner` to `ctx` above so that we only query once and provide to all scripts in this group.